### PR TITLE
Make simplecov/rcov optional

### PR DESCRIPTION
**Pull request by elliotcm**
We don't need to see coverage on every test run, and it takes a second or two every time.

We do want to see it on CI, so re-enable there.  Set COVERAGE=true in your test env vars if you would like to keep the current behaviour.

### Before

```
bin/rspec spec/models/candidate_spec.rb:24  17.05s user 6.01s system 97% cpu 23.701 total
```

### After
```
bin/rspec spec/models/candidate_spec.rb:24  9.75s user 4.84s system 95% cpu 15.233 total
```

## Guidance to review

Could someone please confirm that I've put the CI configuration in the right place, thank you.

## Link to Trello card

https://trello.com/c/0M84STpQ/695-speed-up-app-development

### CODE REVIEW

**Comment by elliotcm** [comment]
[Speed up app development](https://trello.com/c/0M84STpQ/695-speed-up-app-development)

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -14,23 +14,26 @@
...
-command_name += "-#{ENV['TEST_MATRIX_NODE_NAME']}"      if ENV['TEST_MATRIX_NODE_NAME']
-command_name += "-#{ENV['DEFAULT_FEATURE_FLAG_STATE']}" if ENV['DEFAULT_FEATURE_FLAG_STATE']
-command_name += "-#{ENV['TEST_ENV_NUMBER']}"            if ENV['TEST_ENV_NUMBER']
-
-SimpleCov.command_name(command_name)
-
-SimpleCov.start 'rails' do
-  enable_coverage :branch
+
+if ENV.fetch('COVERAGE', 'false') == 'true'
```
Comment (carlosmartinez) [inline spec/spec_helper.rb:18]: perhaps `if ENV.fetch('COVERAGE', 'true') == 'true'` is safer here, then in local development we can just set `COVERAGE` to false. whaddayathink?

Comment (elliotcm) [reply spec/spec_helper.rb:18]: Sure.  Is there somewhere we can put it where it'll be turned off for new developers?

Comment (elliotcm) [reply spec/spec_helper.rb:18]: Does that run in `test`? (which is where the coverage is running)

I guess https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/.env.test instead?

Comment (carlosmartinez) [reply spec/spec_helper.rb:18]: cool, `.env.test` is a good shout

**Review by carlosmartinez — APPROVED** [review]

**Pull request merged**